components/serializer.rst

```diff
@@ -721,6 +721,18 @@ There are several types of normalizers available:
     :phpclass:`DateTime` and :phpclass:`DateTimeImmutable`) into strings.
     By default it uses the RFC3339_ format.
 
+    The normalizer attempts to denormalize all time strings with a default timezone
+    if the defaultContext sets a ``TIMEZONE_KEY`` and ``FORCE_CONTEXT_TIMEZONE`` is
+    set to ``true``.
+
+    PHP :phpclass:`DateTimeInterface` constructor will return a result with a timezone
+    that may not be the default timezone if the time parameter has timezone information
+    embedded (e.g. 2010-01-28T15:00:00+02:00) or the time parameter is a UNIX timestamp
+    (e.g. @946684800). To use the default PHP behavior, set ``TIMEZONE_KEY`` to ``null``.
+
+    Setting ``FORCE_CONTEXT_TIMEZONE`` to ``false`` is considered deprecated from
```

Comment:
```suggestion
    .. deprecated:: 4.2
    
        Setting ``FORCE_CONTEXT_TIMEZONE`` to ``false`` is considered deprecated from
        Symfony 4.2.
```

Reply:
Do have sind problems applying my suggestion?
We need the „.. deprecated:: 4.2“ directive

Reply:
Sorry, only my lack of understanding.  I can do that if you consider it important.

Reply:
Removed Deprecation completely.  No longer germane.